__test__/index.test.js

```diff
@@ -38,27 +38,53 @@ beforeEach(() => {
       description: 'Gettin there',
       size: 10,
       x: 55.11627906976744,
-      y: 44.88372093023257,
```

Comment:
At least keep one Y defined to pass the test below "uses the supplied y value if defined"

Reply:
I've now bought this point back